pkg/controller/clusterpool/clusterpool_controller_test.go

```diff
@@ -323,9 +324,78 @@ func TestReconcileClusterPool(t *testing.T) {
 			expectedInventoryValidStatus: corev1.ConditionFalse,
 			expectedInventoryMessage:     map[string][]string{"BrokenByCloud": {"test-cdc-1"}},
 			expectedCDCReason:            map[string]string{"test-cdc-1": hivev1.CustomizationApplyReasonBrokenCloud},
-			expectInventory:              true,
+			expectedPoolVersion:          inventoryPoolVersion,
 			expectedAssignedCDCs:         map[string]string{"test-cdc-1": "c1"},
 		},
+		{
+			name: "cp with inventory and good cdc is valid, cd created, and calimed",
+			existing: []runtime.Object{
+				inventoryPoolBuilder().Build(testcp.WithSize(1)),
+				testcdc.FullBuilder(testNamespace, "test-cdc-1", scheme).Build(),
+				testclaim.FullBuilder(testNamespace, "test", scheme).Build(testclaim.WithPool(testLeasePoolName)),
+				cdBuilder("c1").Build(
+					testcd.Installed(),
+					testcd.WithClusterPoolReference(testNamespace, testLeasePoolName, "test"),
```

Comment:
Was this one supposed to start off unassigned so we could prove that it does *not* get assigned? (Because it is not running and it is not an "always running" cloud.)

Or is that covered elsewhere? (I didn't look.)